uk.co.compucorp.civicrm.hrleaveandabsences/views/absence-tab/components/absence-tab-entitlements.html

```diff
@@ -1 +1,154 @@
-Entitlements
+<section class="panel panel-default panel-default-plain">
+  <header class="panel-heading">
+    <h2 class="panel-title">Contract Entitlements</h2>
+    <hr/>
```

Comment:
HR tags should not be used to add margins, HR is used only for adding a line. We should follow the markup from styleguide, and this is what I can find from styleguide
```
<div class="panel panel-default">
  <div class="panel-heading">
    <h3 class="panel-title">Panel Default</h3>
  </div>
  <div class="panel-body">
    Panel content
  </div>
  <div class="panel-footer">Panel footer</div>
</div>
```
So it should work without having to add `HR`.